modules/admin_manual/pages/installation/docker/index.adoc

```diff
@@ -96,143 +111,178 @@ ownclouddockerserver_owncloud_1   … /usr/bin/owncloud server  Up      0.0.0.0:
 ownclouddockerserver_redis_1      … /bin/s6-svscan /etc/s6    Up      {std-port-redis}/tcp
 ----
 
-In it, you can see that the database, ownCloud, and Redis containers are running, and that only ownCloud is accessible via port {std-port-http} on the host machine.
+In it, you can see that the database, ownCloud, and Redis containers are running, and that only
+ownCloud is accessible via port {std-port-http} on the host machine.
 
 [IMPORTANT]
 ====
 All files stored in this setup are contained in Docker volumes, rather than a physical filesystem tree.
-It is the admin's responsibility to persist the files.
-Use, e.g., `docker volume ls | grep ownclouddockerserver` to inspect the volumes.
-Use e.g., `docker run -v ownclouddockerserver_files:/mnt ubuntu tar cf - -C /mnt . > files.tar` to export the files as a tar archive.
+It is the admin's responsibility to persist the files. +
+To inspect the volumes run:
+[source,console]
+----
+sudo docker volume ls | grep ownclouddockerserver`
+----
+To export the files as a tar archive run:
+[source,console]
+----
+sudo docker run -v ownclouddockerserver_files:/mnt \
+            ubuntu tar cf - -C /mnt . > files.tar
+----
 ====
 
 [TIP]
 ====
-Although the containers are up and running, it may still take a few minutes until ownCloud is fully functional.
-Run, e.g., `docker-compose logs --follow owncloud` and inspect the log output.
-Wait until the output shows "Starting apache daemon..." before you access the web UI.
+Although the containers are up and running, it may still take a few minutes until ownCloud is fully
+functional. +
+To inspect the log output:
+[source,console]
+----
+sudo docker-compose logs --follow owncloud
+----
+Wait until the output shows **Starting apache daemon...** before you access the web UI.
 ====
 
 [IMPORTANT]
 ====
-Although all important data persists after `docker-compose down; docker-compose up -d`, there are certain details that get lost, e.g., default apps may re-appear after they were uninstalled.
+Although all important data persists after:
+[source,console]
+----
+sudo docker-compose down; docker-compose up -d
+----
+there are certain details that get lost, e.g., default apps may re-appear after they were uninstalled.
 ====
 
 === Logging In
 
-To log in to the ownCloud UI, open `http://localhost:{std-port-http}` in your browser
+To log in to the ownCloud UI, open `pass:a[http://localhost:{std-port-http}]` in your browser
 of choice, where you see the standard ownCloud login screen, as in the
 image below.
 
 image:docker/owncloud-ui-login.png[The ownCloud UI via Docker]
 
-The username and password are the admin username and password which you
-stored in `.env` earlier. Note that these will not change between deploys
-even if you change the values in .env. 
+The username and password are the admin username and password which you stored in `.env` earlier.
```

Comment:
> are the admin username and password 
are the credentials? Sounds better.

Reply:
I changed the original text according your suggestion